supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+[[partner-and-hybrid-cloud-infrastructure-conventions]]
+
+[discrete]
+[[access-mode]]
+==== image:images/yes.png[yes] access mode (noun)
+*Description*: The concurrency method used when accessing a storage component, such as `readwriteonce` or `readwritemany`.
+
+*Use it*: Yes
+
```

Comment:
Even if an entry doesn't have a value for them, the "Incorrect forms" and "See also" labels should be there. Would you mind going through all of the new entries to make sure they're there?

Reply:
Ah, my apologies, my impression was that those could be omitted if they were not used. I'll add those.